Documentation (#320)
IBM/ibm-block-csi-driver: docs/content/troubleshooting/csi_ug_troubleshooting_logs.md

# Log collection

Use the CSI (Container Storage Interface) driver logs for problem identification.

**Note:** These procedures are applicable for both Kubernetes and Red Hat® OpenShift®. For Red Hat OpenShift, replace `kubectl` with `oc` in all relevant commands.

To collect and display logs, related to the different components of IBM® block storage CSI driver, use the following Kubernetes commands:

> ## Log collection for CSI pods, daemonset, and StatefulSet
> 
> `kubectl get all -n <namespace>  -l csi`

Reviewer: this one isn't about logs
```suggestion
## name collection of CSI pods, daemonset, and StatefulSet

`kubectl get all -n <namespace> -l csi`
```

> To collect and display logs, related to the different components of IBM® block storage CSI driver, use the following Kubernetes commands:
> 
> ## Log collection for CSI pods, daemonset, and StatefulSet
> 
> `kubectl get all -n <namespace>  -l csi`

Reviewer: this one isn't about logs
```suggestion
To collect and display status and logs, related to the different components of IBM® block storage CSI driver, use the following Kubernetes commands:

## Status collection of CSI pods, daemonset, and statefulset

`kubectl get all -n <namespace> -l csi`
```

Author: @oriyarde @barakda  if this isn't log collection should I
1. move this status collection to the misc troubleshooting section?
or
2. rename the whole section to just "log and status" collection?

Reviewer: I prefer 2.

also when we ask for "log collection", we mean status as well.
a common term for all of these could be "debug information"

Author: DONE

Reviewer: it doesn't seem to be done here, but it was [indeed done in 1.7.0](https://github.com/IBM/ibm-block-csi-driver/pull/360/files/5bdd9dcccb1b39f1b6fc768275a981cc6a51eef6#diff-2e0d02fce1320e366e845f71164eb56f5ee63fe6074dc8aecb9c866fe5ed6886L9-R9). resolving
